File: ActiveProbe.Utils/ViewModel/Zone.cs
using System.Collections.Generic;
namespace ActiveProbe.Utils.ViewModel
{
    public class ZoneVm
    {
        public int ZoneId { get; set; }
        public string Title { get; set; }
        public string Type { get; set; }
        public bool IsActive { get; set; }
    }
    public class ZonePointVm
    {
        public int Id { get; set; }
        public int ZoneId { get; set; }
        public double Lat { get; set; }
        public double Lon { get; set; }
    }
    public class ZoneKmlVm
    {
        public int Id { get; set; }
        public int ZoneId { get; set; }
        public string KmlFile { get; set; } //Url
    }
    public class ZPoint
    {
        public int ZoneId { get; set; }
        public string Title { get; set; }
        public string Type { get; set; }
        public double Lat { get; set; }
        public double Lon { get; set; }
    }
}

File: ActiveProbLTE/ClientApp/src/app/Shared/models/Zone.ts
export class Zone {
  constructor(public ZoneId: number, public Title: string, public Type: string, public IsActive: boolean) { }

}
export class point {
  constructor(public ZoneId: number, public Lat: number, public Lon: number) { }
}
// export interface Izone{
//      TItle:string;
//      Type:string;
//      IsActive:boolean;
// }
// export interface Ipoint{
//     ZoneId:number;
//     Lat:string;
//     Lon:string;
// }
